packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -627,23 +640,16 @@ async function warnIfDependenciesRequireAdditionalSetupAsync(
     skipSDKVersionRequirement: true,
   });
 
-  const extraSetupPath = projectHasModule('expo/requiresExtraSetup.json', projectRoot, exp);
```

Comment:
Ensure the following works without throwing:
- in a managed project delete the node_modules (add the sdkVersion to the app.json)
- `expo eject --no-install`

Reply:
yeah it works, although if you use `app.config.js` instead of app.json it doesn't:
```
Failed to read config at: /path/to/app.config.js
```

but that's also the case on master

Reply:
good find, opened a PR to fix